frontend/profil.js

```diff
@@ -8,6 +9,7 @@ function toUser(nom, prenom, date, email, password) {
     };
 }
 
+//Get the user input in the form
 function getUser() {
```

Comment:
toUser just useless remove it and do a function getUser(){
return {email: email,
        nom: nom,
        prenom: prenom,
        date: date,
        password: password,
    };
}
as in food.js